src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -921,4 +921,7 @@ nuget client-certs List</value>
   <data name="Option_Help" xml:space="preserve">
     <value>Show command help and usage information.</value>
   </data>
+  <data name="SourcesCommandProtocolVersionDescription" xml:space="preserve">
+    <value>The NuGet server protocol version to be used. The current version is "3". Supported in NuGet 3.0+.</value>
```

Comment:
```suggestion
    <value>The NuGet server protocol version to be used. The default version is "3". Supported in NuGet 3.0+.</value>
```

Reply:
I've changed the wording of this description to better match the behaviour of code
- "2" is the default version which is not being written in xml
- "3" is called 'current' as per [nuget.config File Reference](https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file#packagesources)